Fastlane/.build.yml

```diff
@@ -26,7 +28,22 @@ alpha:
   <<: *DEFAULTS
   app:
     <<: *APP_DEFAULTS
+    scheme: GovUKAlpha
```

Comment:
Is it worth changing the default scheme to `GovUKDev`?

Reply:
Tested with running fastlane unit locally and things working okay as is, so as discussed we can leave for now